charts/nginx-ingress/values.yaml

```diff
@@ -487,8 +487,10 @@ controller:
   ## Enable dynamic reloading of certificates
   enableSSLDynamicReload: true
 
-  ## Enable telemetry reporting
-  enableTelemetryReporting: true
+  ## Configure telemetry reporting options
+  telemetryReporting:
```

Comment:
[the schema](charts/nginx-ingress/values.schema.json) needs to be updated along with this change